### PR TITLE
New package: nvidia-vaapi-driver-0.0.9

### PR DESCRIPTION
**Pull request by someone13574**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64)

### CODE REVIEW

File: srcpkgs/nvidia-vaapi-driver/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'nvidia-vaapi-driver'
+pkgname=nvidia-vaapi-driver
+version=0.0.7
+revision=1
+archs="x86_64"
+build_style=meson
+hostmakedepends="pkg-config libglvnd-devel libva-devel gst-plugins-bad1-devel"
```
Comment (abenson) [inline srcpkgs/nvidia-vaapi-driver/template:7]: -devel should go in `makedepends`

File: srcpkgs/nvidia-vaapi-driver/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+makedepends="nv-codec-headers"
+depends="nvidia gst-plugins-bad1"
```
Comment (abenson) [inline srcpkgs/nvidia-vaapi-driver/template:9]: This whole line can be removed; `gst-plugins-bad1` gets auto-detected as well as `libEGL`, one you didn't explicitly list.  `nvidia` shouldn't be a hard-requirement, as there is a plan to also package `nvidia-open` at some point.

```
=> nvidia-vaapi-driver-0.0.7_1: running pre-pkg hook: 999-collected-rdeps ...
   glibc>=2.32_1 gst-plugins-bad1>=1.18.3_2 libEGL>=7.11_1
```

**Comment by abenson** [comment]
Does it work with `nvidia470`?

**Comment by Owenlaw222** [comment]
Works with `nvidia470`

**Comment by Owenlaw222** [comment]
Marked as draft due to issue with Nvidia 525 drivers.

https://github.com/elFarto/nvidia-vaapi-driver/issues/126

**Pull request merged**